packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1505,6 +1505,11 @@ export class GarbageCollector implements IGarbageCollector {
         // For summarizer client, queue the event so it is logged the next time GC runs if the event is still valid.
         // For non-summarizer client, log the event now since GC won't run on it. This may result in false positives
         // but it's a good signal nonetheless and we can consume it with a grain of salt.
+        // Inactive errors are usages of Objects that are unreferenced for a period of 7 days.
+        // SweepReady errors are usages of Objects that will be deleted by GC Sweep!
+        // Events generated:
+        // InactiveObject_Loaded, InactiveObject_Changed, InactiveObject_Revived
```

Comment:
This function only logs the *_Loaded events. You should update the comment to reflect that.
`logUnreferencedEvents` is the one that logs all the events so this comment belongs there.